examples/hosted-delivery-function-templates/javascript/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "@contentful/node-apps-toolkit": "2.2.0"
```

Comment:
nit: if this library is only used for types, might also be ok to only add it in devDependencies